externs/olcsx.js

```diff
@@ -24,6 +24,12 @@ olcsx.OLCesiumOptions;
  */
 olcsx.OLCesiumOptions.prototype.map;
 
+/**
+ * Allows the passing of property value to the `Cesium.Scene`.
+ * @type {Object|undefined}
```

Comment:
You must use a proper definition of the options instead of `Object`. The definition should be added to Cesium.externs.js.